server.go

```diff
@@ -294,6 +309,17 @@ func makeUDPBuffer(size int) func() interface{} {
 func (srv *Server) init() {
 	srv.queue = make(chan *response)
 
+	const workersCount = 100
+	for i := 0; i < workersCount; i++ {
+		go func() {
```

Comment:
These goroutines are unnamed and won’t appear clearly in traces (`Server.init.func1`). They should be a seperate named function or method.